src/app/containers/PageHandlers/withData.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { shape, element } from 'prop-types';
+import articlePropTypes from '../../models/propTypes/article';
+import ErrorMain from '../ErrorMain';
+import nodeLogger from '../../helpers/logger.node';
+import deepGet from '../../helpers/json/deepGet';
+
+const logger = nodeLogger(__filename);
+
+// checks for data and status keys, setting default status if not found
+const constructRenderObject = data => ({
+  status: deepGet(['status'], data) ? data.status : 500,
+  assetData: deepGet(['data'], data),
+});
+
+// checks for assetData and 200 status
+const shouldRender = data => {
+  let hasDataAnd200Status = false;
+  const { status, assetData } = constructRenderObject(data);
+
+  if (assetData && status === 200) hasDataAnd200Status = true;
```

Comment:
```suggestion
  if (assetData && status === 200) {
    hasDataAnd200Status = true;
  }
```

Reply:
Any particular reason for this or is it just preference? This just seems like a cleaner way to simply set a variable.

Reply:
Cleaner would be 
```
hasDataAnd200Status = assetData && status === 200
```
Im not a fan of if statements without a block, theyre so rare they just tend to be confusing imo